Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php

```diff
@@ -139,7 +139,7 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt)
                 continue;
             }
 
-            if ($token->equals(',')) {
+            if ($token->equals(',') && !in_array($tokens[$index - 1]->getContent(), array(')', ']'), true)) {
```

Comment:
Sorry I don't understand what you want to test exactly.

Please provide complete test data if you want me to add.